### DISCUSSION TITLE
Add new device class for sensors: version

### DISCUSSION THREAD

**Discussion by dougiteixeira** [home-assistant/architecture#733] [Entity Models] [answered]
# Context:
In PR https://github.com/home-assistant/core/pull/66365 to add icons to the version sensors of the `hassio` integration, it was found that there is no standard between the integrations.

### Current overview:
Integration | Icon
-- | --
`apcupsd` | `mdi:information-outline`
`dsmr_reader` | `mdi:alert-circle`
`edl21` | `mdi:flash`
`ezviz` | `none`
`google_wifi` | `mdi:checkbox-marked-circle-outline` and `mdi:update`
`hassio` | `none`
`homewizard` | `mdi:counter`
`minecraft_server` | `mdi:numeric`
`system_bridge` | `mdi:counter`
`tasmota` | `none`
`version` | `mdi:package-up`

So it was scored by @ludeeus if it wasn't time to create a specific device class for version sensors that standardize the icon.

This discussion aims to define that.

# Proposal:
The proposal is to create the `version` device class for sensors, using the `mdi:information-outline` icon.

Some interfaces (`google_wifi`, `hassio`, `version`, etc) have sensors for the current version and the latest version. Would it be necessary to distinguish these two sensors?

### Decision points:
1. Creation of the `version` device class.
2. Icon definition for the new device class.
3. Definition of the need to distinguish current and latest version sensors.

**Comment by frenck** [comment, marked as answer]
This one has been around for some time, yet, we've quickly discussed it.

Between the original suggestion and now, a lot changed. Device can list versions now (and those attributes can be queried in templates) and update entities also provide this information.

A quick scan on the codebase, showed that almost all migrated to these solutions at this point.

Therefore, I'm going to close this one up, as other alternatives have been put into place.

../Frenck
